### PR TITLE
bug: fix config load

### PR DESCRIPTION
**Pull request by BohuTANG**
## Summary

Summary about this PR

Close #issue

### CODE REVIEW

**Comment by Xuanwo** [comment]
`#[serde(default)]` is required for all config struct, otherwise, load from file will failed.

File: common/configs/src/eth.rs [hunk 1]
```diff
@@ -27,13 +27,14 @@ use serfig::parsers::Toml;
 use crate::LogConfig;
 use crate::StorageConfig;
 
-#[derive(Parser, Debug, Default, Clone, Serialize, Deserialize)]
+#[derive(Parser, Debug, Clone, Serialize, Deserialize)]
```
Comment (Xuanwo) [inline common/configs/src/eth.rs:30]:
```suggestion
#[derive(Parser, Debug, Clone, Serialize, Deserialize)]
#[serde(default)]
```

All config should have `#[serde(default)]` to make loading from files work.

File: common/configs/src/eth.rs [hunk 2]
```diff
@@ -27,13 +27,14 @@ use serfig::parsers::Toml;
...
 pub struct ExportConfig {
     #[clap(
         short = 'p',
         long,
         value_parser,
-        default_value = "https://127.0.0.1:8845",
+        default_value_t,
+        default_value = "127.0.0.1:8845",
```
Comment (Xuanwo) [inline common/configs/src/eth.rs:37]:
```suggestion
        default_value = "https://127.0.0.1:8845",
```

Should be exactly the same with Default export config.

File: common/configs/src/log.rs [hunk 3]
```diff
@@ -17,13 +17,22 @@ use clap::Parser;
 use serde::Deserialize;
 use serde::Serialize;
 
-#[derive(Parser, Debug, Default, Clone, Serialize, Deserialize)]
+#[derive(Parser, Debug, Clone, Serialize, Deserialize)]
```
Comment (Xuanwo) [inline common/configs/src/log.rs:20]:
```suggestion
#[derive(Parser, Debug, Clone, Serialize, Deserialize)]
#[serde(default)]
```

**Comment by BohuTANG** [comment]
Thanks @Xuanwo !

**Pull request merged**